manifests/config.pp

```diff
@@ -51,6 +51,8 @@
   $truststore                  = $rundeck::truststore,
   $truststore_password         = $rundeck::truststore_password,
   $user                        = $rundeck::user,
+  $security_roles_array_enabled             = $rundeck::security_roles_array_enabled,
+  $security_roles_array                     = $rundeck::security_roles_array,
```

Comment:
could you check this please? Maybe you've used hard tabs to align it, please try it with whitespace/soft tabs.